Framework/Algorithms/src/RebinByPulseTimes.cpp

```diff
@@ -60,11 +60,11 @@ void RebinByPulseTimes::doHistogramming(IEventWorkspace_sptr inWS,
     el.generateHistogramPulseTime(*XValues_new, y_data, e_data);
```

Comment:
@SimonHeybrock  this would require a lot of changes to EventList. I think it should be handled in another issue.